TodoTests/XTests.swift

```diff
@@ -11,10 +11,16 @@ import XCTest
 @testable import Todo //DONT FORGET
 
 
+
 class XTests: XCTestCase {
     
-    func testThatSomething() {
+    let td = TodoViewModel()
+    
+    func testTransformedData() {
+        
+        let s = td.transformedDatasource("Jerry")
         
+        XCTAssert(s[0] == "Jerry:")
```

Comment:
Agh! Embarrassingly enough, I had the email notifications for this repo filtered out with the billion other repos I'm subscribed to, hence the delay. Sorry about dropping the ball here, will pick this back up!